portfolio/src/main/webapp/script.js

```diff
@@ -127,7 +127,11 @@ function createListTextElement(text) {
 /** Creates an <li> element containing an image. */
 function createListImageElement(imageURL) {
     const liElement = document.createElement('li');
-    liElement.innerHTML = '<img src="' + imageURL + '" class="list-image">';
+    const imgElement = document.createElement('img');
+    imgElement.setAttribute('src', imageURL);
+    imgElement.setAttribute('width', '200px');
```

Comment:
class can still be set dynamically (and should be, so that all styles are kept in CSS)

Reply:
done